p751toolbox/consts.go

```diff
@@ -14,3 +16,52 @@ var Affine_yPB = PrimeFieldElement{A: Fp751Element{0xd1e1471273e3736b, 0xf9301ba
 
 // The value of (a+2)/4 for the starting curve E_0 with a=0: this is 1/2
 var E0_aPlus2Over4 = PrimeFieldElement{A: Fp751Element{0x124d6, 0x0, 0x0, 0x0, 0x0, 0xb8e0000000000000, 0x9c8a2434c0aa7287, 0xa206996ca9a378a3, 0x6876280d41a41b52, 0xe903b49f175ce04f, 0xf8511860666d227, 0x4ea07cff6e7f}}
+
+const (
```

Comment:
Everything from this point until end of the file must be removed - it is already in "sidhp751/sidhp751.go"